main31.js

```diff
@@ -1,73 +1,84 @@
+import {
+  CLOSE_ICON,
+  MESSAGE_ICON,
+  TICK_ICON,
+  style,
+  setColorsFromThemeName,
+} from "./assets15.js";
 
 let loadInterval;
 var chatbotID = "";
-//const BASE_URL = "http://localhost:5001/dev/";
-const BASE_URL = "https://sswj8m0la3.execute-api.af-south-1.amazonaws.com/dev/";
+const BASE_URL = "http://localhost:5001/dev/";
+//const BASE_URL = "https://sswj8m0la3.execute-api.af-south-1.amazonaws.com/dev/";
 let messageWidget;
 var chatbotInfo = {};
 
 function sendMessage() {
   const inputField = document.getElementById("penpal-chatbot-widget-input");
   const input = inputField.value.trim();
-  (input != "" && chatbotID) && handleMessageSend(input);
+  input != "" && chatbotID && handleMessageSend(input);
   inputField.value = "";
 }
 
 // Function to receive and process data from the page
-window.setDataFromPage = function(data) {
+window.setDataFromPage = function (data) {
   // Process the data as needed
   chatbotID = data.id;
   //chatbotID = "646330d6c251f7689abd9eb8";
   getChatbotInfo();
 };
 
 const getChatbotInfo = async () => {
-  if(!chatbotID){
+  if (!chatbotID) {
     console.log("No chatbot ID found.");
     return;
   }
-  const url = BASE_URL + "chatbot/get_public_info"
+  const url = BASE_URL + "chatbot/get_public_info";
   const requestInfo = {
     chatbotID: chatbotID,
-  };    
-  console.log("requesting chagbot")
+  };
+  console.log("requesting chagbot");
 
   // fetch chatbot info
   await fetch(url, {
     method: "POST",
     headers: {
       "Content-Type": "application/json",
-      },
-      body: JSON.stringify(requestInfo),
-    }).then(res => res.text())
-      .then(body => {
-        try {
-          const bodyJson = JSON.parse(body);
-          console.log("chatbot info: ", bodyJson);
-          chatbotInfo = bodyJson;
-          document.getElementsByClassName("penpal-header")[0].innerHTML = chatbotInfo.chatTitle;
-          document.getElementsByClassName("penpal-first-message")[0].innerHTML = chatbotInfo.welcomeMessage;
-          setColorsFromThemeName(chatbotInfo.colorScheme);
-
-          // need to inject styles after setting colors
-          messageWidget.injectStyles();
-          messageWidget.setIsVisible(true);
-        } catch {
-            console.log("Error fetching");
-            throw Error(body);
-        }
-      }).catch(error => {
-        console.log("Error fetching chatbot info:", error);
-      });
-}
+    },
+    body: JSON.stringify(requestInfo),
+  })
+    .then((res) => res.text())
+    .then((body) => {
+      try {
+        const bodyJson = JSON.parse(body);
+        console.log("chatbot info: ", bodyJson);
+        chatbotInfo = bodyJson;
+        document.getElementsByClassName("penpal-header")[0].innerHTML =
+          chatbotInfo.chatTitle;
+        document.getElementsByClassName("penpal-first-message")[0].innerHTML =
+          chatbotInfo.welcomeMessage;
+        //setColorsFromThemeName(chatbotInfo.colorScheme);
+        setColorsFromThemeName("dark");
```

Comment:
The line above should not be commented out, as we want to set the theme to the one fetched from the DB, rather than being hardcoded.